app/views/pages/_modal-art.html.erb

```diff
@@ -0,0 +1,72 @@
+ <div class="container justify-content-center my-4">
+
+    <div class="container">
+
+        <div class="container art-piece-grid">
+          <% @user.art_pieces.each do |art_piece| %>
+        <div class="card-art-piece">
+
+          <% if art_piece.photos.attached? %>
+            <%= cl_image_tag art_piece.photos[0].key, crop: :fill, class:"img-index" %>
+          <% else %>
+            <img class="img-index" src="https://images.unsplash.com/photo-1581783342308-f792dbdd27c5?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1834&q=80"/>
+          <% end %>
+
+          <div class="card-art-piece-infos">
+          
+            <div>
+
+              <h2><%= art_piece.title %></h2>
+              <p><%= art_piece.artist %>.</p>
+
+                <div class="functional-buttons">
+                  <h2><%= link_to 'Delete', art_piece, method: :delete, data: { confirm: 'Are you sure?' } %></h2>
```

Comment:
Should we use a fontawesome button for delete? 
<i class="fas fa-trash"></i>
https://fontawesome.com/icons/trash?style=solid